Dockerfile

```
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+WORKDIR /app
+COPY package.json .
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y nodejs npm
+RUN npm install
```

As I can see you have yarn.lock file in your repo. So I guess it's better to use it in Docker to install dependencies as well.
In this case we can use the lock file:
```suggestion
RUN yarn install --frozen-lockfile
```

Otherwise, prefer to use `npm ci` instead of simple 'install' to avoid issues with dependencies.